frontend/src/App.tsx

```diff
@@ -1,14 +1,21 @@
 import React from "react";
 import { useState } from "react";
 import Navbar from "./components/Navbar";
+import {
+  RecoilRoot,
+  atom,
+  selector,
+  useRecoilState,
+  useRecoilValue,
+} from "recoil";
 
 function App() {
   const [count, setCount] = useState(0);
 
   return (
-    <div className="App">
+    <RecoilRoot>
```

Comment:
You should move these to surround `<App />` in `main.tsx` instead so that they cover everything